packages/react-scripts/config/webpack.config.js

```diff
@@ -177,6 +198,11 @@ module.exports = function(webpackEnv) {
       // changing JS code would still trigger a refresh.
     ].filter(Boolean),
     output: {
+      // Prevents conflicts when multiple Webpack runtimes (from different apps)
+      // are used on the same page.
+      //jsonpFunction: `webpackJsonp${appPackageJson.name}`,
+      jsonpFunction: camelCase(appPackageJson.name + 'JsonpCallback'),
```

Comment:
why move the jsonpFunction?

Reply:
I caused a weird error with building and it seemed that order mattered in order to fix it

Reply:
Try moving it again and test

Reply:
Moved back and all still works